static/components/PoiCard.js

```diff
@@ -36,16 +36,22 @@ class PoiCard extends React.Component {
     return Math.trunc(R * c)
   }
 
-  hoverCard() {
-    this.base.classList.toggle('current-card')
-    const marker = document.getElementById(this.props.details._id)
-    marker.classList.toggle('selected')
-  }
+  // hoverCard() {
```

Comment:
don't keep commented code, just delete it (git is there if you ever need it back)